src/code/z_play_hireso.c

```diff
@@ -1,27 +1,1349 @@
 #include "global.h"
+#include "z64bombers_notebook.h"
+#include "interface/schedule_static/schedule_static.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_play_hireso/func_8016AC10.s")
+// TODO: Create xml when build supports yaz0 compression/decompression
```

Comment:
Should make identical comments for other places that is waiting for the `yaz0` scripts, not a bit deal tho

Reply:
Where else do we need yaz0 scripts? I know map has some stuff, but that is still in a draft state #1032.

Reply:
`gItemIcons` is a big one, that gets used in parameter and kaleido. 

In Kaleido Collect:
```c
extern TexturePtr D_08061000; // gBombersNotebookIconTex
extern TexturePtr D_08062000; // gSongNoteIconTex
```
In Kaleido Map:
```c
extern TexturePtr D_09007500; // gPlayerFaceIcon
```
In Parameter
```c
extern TexturePtr D_08095AC0; // gMagicArrowEquipEffectTex
```

Really just items and pause stuff, so maybe it felt like more just because that's what I work on. I can always just add comments later to these above, not a big deal here

Reply:
Sounds good, but I'm going to change it to just `// TODO: Needs YAZ0`. That way it would be easy to search for all of them if they are all the same.